kong/cmd/start.lua

```diff
@@ -18,7 +18,10 @@ local function execute(args)
   local dao = assert(DAOFactory.new(conf))
   xpcall(function()
     assert(prefix_handler.prepare_prefix(conf, args.nginx_conf))
-    assert(dao:run_migrations())
+    if args.run_migrations then
+      assert(dao:run_migrations())
+    end
+    assert(dao:are_migrations_uptodate())
```

Comment:
Instead of throwing the error, I think it would be nice if it would retry for a specified timeout (default 60 seconds?). That would allow orchestration tools to run a single `kong migrations up` and then schedule Kong containers/nodes using `kong start`. The containers will then wait before actually starting until migrations are complete, instead of failing.